exercises/concept/squeaky-clean/.docs/hints.md

```diff
@@ -0,0 +1,35 @@
+# Hints
+
+## 1. Replace any spaces encountered with underscores
+- [This tutorial][string-tutorial] is useful.
+- [Reference documentation][string-docs] for `String`.
+- `String` provides a [replace][replace] function.
+
+## 2. Replace control characters with the upper case string "CTRL"
+- The common control characters are: \n, \t, and \r.
```

Comment:
You mention them specifically in the instructions (which is good), so there is no need to repeat it here